osu.Game/Rulesets/Objects/Legacy/Osu/ConvertHitObjectParser.cs

```diff
@@ -46,12 +61,13 @@ protected override HitObject CreateSlider(Vector2 position, bool newCombo, int c
 
         protected override HitObject CreateSpinner(Vector2 position, bool newCombo, int comboOffset, double endTime)
         {
+            forceNewCombo |= FormatVersion <= 8 || newCombo;
```

Comment:
add a comment here (similar to the PR description). will make it much easier for people reading through these legacy decoder methods to know exactly what is going on.